app/controllers/tasks_controller.rb

```diff
@@ -0,0 +1,41 @@
+class TasksController < ApplicationController
+  
+  def index
+    @tasks = Task.all
+    render json: @tasks
+  end
+
+  def show
+    @task = Task.find(params[:id])
+  end
+
+  def create
+    @task = Task.new(task_params)
+    if @task.valid?
```

Comment:
`.save!` will run validations in this case. you can save yourself a line here.

See: http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-save-21